integration_tests/src/main/python/sort_test.py

```diff
@@ -228,12 +229,12 @@ def test_large_orderby(data_gen, stable_sort):
 # This is similar to test_large_orderby, but here we want to test some types
 # that are not being sorted on, but are going along with it
 @pytest.mark.parametrize('data_gen', [byte_gen,
-    string_gen,
-    float_gen,
-    date_gen,
-    timestamp_gen,
-    decimal_gen_default,
-    StructGen([('child1', byte_gen)]),
+    # string_gen,
```

Comment:
If they were no longer needed, better to remove them.

Reply:
Agree if we don't think we need these any more then we should delete them instead of commenting them out. But that said are we removing this because of performance? If so then you could refactor it so there is a single test with one column for each type we want to test and do a single join. Was it commented out just because you were testing? It would be nice to know why these tests are being removed so we can make a proper decision on how to deal with it.

Reply:
These tests are necessary. I just forgot to uncomment them after the debugging work on Array of Byte. I am sorry for making you confused  :(